composer.json

```diff
@@ -22,6 +23,7 @@
       "php": "7.1"
     },
     "preferred-install": {
+        "woocommerce/action-scheduler": "source",
```

Comment:
Should we perhaps also use dist here?

Reply:
@kloon Thanks :) Yes, I'll update that to `dist`. I'm doing a similar PR https://github.com/woocommerce/woocommerce-admin/pull/3557. I'll leave this one on needs feedback for now.

Reply:
Done in `9b99f3b`.